src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -216,6 +216,36 @@ Extension types are also used to define the type of commutative triangles:
         t₂ ≡ t₁ ↦ h t₂]   -- the diagonal is exactly `h`
 ```
 
+## Free arrow types
+
+A free arrow in `A` can be described as a triple consisting of two endpoints and
+the arrow between them. This is essentially a special case of
+`cofibration-composition`, but invoking it would require passing through the
+identification `(product A A) ≃ (∂Δ¹ → A)`. Hence it is faster to just show it
+directly.
+
+```rzk
+#def free-arrows
+  ( A : U)
+  : U
+  :=
+    Σ ((a,b) : product A A), hom A a b
+
+#def free-arrow
```

Comment:
This is already formalized in `discrete-types` as [fibered-arr-free-arr](https://rzk-lang.github.io/sHoTT/simplicial-hott/07-discrete.rzk/#define:fibered-arr-free-arr)

Reply:
I see, I had missed the relevant part of `segal-types`, where some of it is already defined.
I will batch it all there and put it under its proper header.
I would like to also keep the one I defined, since it is uncurried. That means that one does not have to invoke `total-equiv-family-of-equiv` twice.